intro-HPC/ch_running_batch_jobs.tex

```diff
@@ -40,6 +40,331 @@ \chapter{Running batch jobs}
+system administrators). However, the user can (and probably should, to avoid
+surprises when newer versions are installed) specify a particular version,
+e.g.,
+
+\begin{prompt}
+%\shellcmd{module load \emph{secondexample/2.7-intel-2018a}}%
+\end{prompt}
+
+Obviously, you need to keep track of the modules that are currently
+loaded. If you executed the two load commands stated above, you will get the
+following:
+
+\begin{prompt}
+%\shellcmd{module list}%
+Currently Loaded Modulefiles:
+  1) examplemodule/1.2.3-intel-2018a        3) intel/2018a
+  2) secondexample/2.7-intel-2018a          4) examplelib/1.2-intel-2014b
+\end{prompt}
+
+
+It is important to note at this point that other modules (e.g., \lstinline|intel/2014a|)
+are also listed, although the user did not explicitly load them. This is
+because \lstinline|secondexample/2.7-intel-2018a| depends on it (as indicated in its name),
+and the system administrator specified that the \lstinline|intel/2014a| module should
+be loaded whenever the \lstinline|secondexample| module is loaded. There are advantages and
+disadvantages to this, so be aware of automatically loaded modules whenever
+things go wrong: they may have something to do with it!
+
+In fact, an easy way to check the components and version numbers of those
+components of a toolchain is to simply load the toolchain and then list the
+modules that are loaded.
+
+To unload a module, one can use the \lstinline|module unload| command. It works
+consistently with the load command, and reverses the latter's effect. However,
+the dependencies of the package are NOT automatically unloaded; the user shall
+unload the packages one by one. One can however unload automatically loaded
+modules manually, to debug some problem. When the \lstinline|secondexample| module is unloaded,
+only the following modules remain:
+
+\begin{prompt}
+%\shellcmd{module unload \emph{secondexample}}%
+%\shellcmd{module list}%
+Currently Loaded Modulefiles:
+  1) examplemodule/1.2.3-intel-2018a        3) intel/2018a
+  2) examplelib/1.2-intel-2014b
+\end{prompt}
+
+Notice that the version was not specified: the module system is sufficiently
+clever to figure out what the user intends. However, checking the list of
+currently loaded modules is always a good idea, just to make sure \ldots
+
+\subsection{Purging all modules}
+\label{subsec:purging-modules}
+
+In order to unload all modules at once, and hence be sure to start in a clean
+state, you can use:
+
+\begin{prompt}
+%\shellcmd{module purge}%
+\end{prompt}
+
+However, on some VSC clusters you may be left with a very empty list of available
+modules after executing \lstinline|module purge|. On those systems, \lstinline|module av| will show you a list of
+modules containing the name of a cluster or a particular feature of a section of
+the cluster, and loading the appropriate module will restore the module list
+applicable to that particular system.
```

Comment:
This needs to be rewritten. In Ghent, `module purge` is always safe, the `cluster` module will not get unloaded (because it's a "sticky" module), only `module --force purge` will get you in trouble (no need to mention that).